### PR TITLE
Javascript2 week2

### PR DESCRIPTION
**Pull request by FurkannOzbek**
- Added List grid and search function 
- Added arrow functions

### CODE REVIEW

**Comment by FurkannOzbek** [comment]
Search function could be more detailed for precise result

File: index.js [hunk 1]
```diff
@@ -48,42 +53,39 @@ function convertCurrency(event) {
   const inputAmount = parseFloat(amount.value);
 
   // Get the exchange rates for the selected currencies
-  const rateFrom = currencies.rates[selectedFromCurrency].rate;
-  const rateTo = currencies.rates[selectedToCurrency].rate;
-
+  const rateFrom = currencies.rates.find((rate) => rate.code === selectedFromCurrency).rate; //
```
Comment (magdazelena) [inline index.js:56]: niceeee, another array function properly used!

File: index.js [hunk 2]
```diff
@@ -48,42 +53,39 @@ function convertCurrency(event) {
...
-  // Add the new currency to the currencies object
-  if (!currencies.rates[newCurrencyCode]) {
-    currencies.rates[newCurrencyCode] = {
-      rate: newRate,
-      text: newCurrencyName,
-    };
-    alert(
-      `New currency ${newCurrencyCode} - ${newCurrencyName} added with rate ${newRate}`
-    );
+  const currencyExists = currencies.rates.some((rate) => rate.code === newCurrencyCode);
```
Comment (magdazelena) [inline index.js:85]: 🤩

**Review by magdazelena — APPROVED** [review]
this looks soo good! Props for using other array functions as well. Your code is really nicely legible and structured, keep up the good work :)

**Pull request merged**